website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -39,12 +39,13 @@ Go to the workspace's notifications settings and press the "Add Notification" bu
 
 Fill in the following fields to configure your notifications:
 
-* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
+* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
```

Comment:
Also --> Is that the right bookmark? It doesn't seem to be formatted with the traditional #, which is why I'm asking!  I think it could make sense to change it to notification payloads, as I have below. Let me know if you disagree!

```suggestion
* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Refer to [Notification Payloads][#notification-payloads] for details.
```
```suggestion
* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
```

Reply:
the [inpage-payload] thing also works because we define the `[inpage-payload]` anchor here https://github.com/hashicorp/terraform-docs-common/blame/main/website/docs/cloud-docs/workspaces/settings/notifications.mdx#L81

Im not sure if there is a preferred way, but I also prefer just [link text](#heading-name), @tylerwolf can jump in and correct me if there is a reason to do it the [link text][anchor defined elsewhere] way

Reply:
okay! i'm okay with you using that anchor (thank you for explaining!) but I would like to keep the Refer to notification payloads for details. 

Where notification payloads is the link text. This adheres to our style guide --> We want to have the most descriptive link text possible and we want to avoid saying "below" when we can.

Reply:
For sure! I think that is should be correct now https://github.com/hashicorp/terraform-docs-common/pull/19/files#diff-84d2bf59be9dd9dd7a529cb93603bfedc4529a645c1f5d8346f76e629ddfb9b6R42

`Refer to [Notification Payloads](#notification-payloads) for details.`